gtsfm/bundle/bundle_adjustment.py

```diff
@@ -61,12 +63,56 @@
 logger = logger_utils.get_logger()
 
 
-class BundleAdjustmentOptimizer:
+class TwoViewBundleAdjustmentProcess(GTSFMProcess):
+    """Dummy class to get the UI graph correct.
+
+    BundleAdjustmentOptimizer is used in two separate places (optimizing
+    between two images and then later optimizing globally), so a separate class
+    is needed due to the way GTSFMProcess is designed.
+    """
+
+    def get_ui_metadata() -> UiMetadata:
+        """Returns data needed to display this process in the process graph."""
+
+        return UiMetadata(
+            "Two-View Bundle Adjustment",
+            "Two-View Estimator",
+            (
+                "Relative Rotations",
+                "Relative Translations",
+                "Triangulated Points",
+                "Relative Pose Priors",
+                "Camera Intrinsics",
+                "Keypoints",
+                "Verified Correspondences",
+            ),
+            ("Optimized Relative Rotations", "Optimized Relative Translations", "Inlier Correspondences"),
+        )
+
+
+class BundleAdjustmentOptimizer(GTSFMProcess):
     """Bundle adjustment using factor-graphs in GTSAM.
 
     This class refines global pose estimates and intrinsics of cameras, and also refines 3D point cloud structure given
     tracks from triangulation."""
 
+    def get_ui_metadata() -> UiMetadata:
+        """Returns data needed to display this process in the process graph."""
+
+        return UiMetadata(
+            "Global Bundle Adjustment",
```

Comment:
same note here and throughout about keyword args, to explain what these args mean